### ISSUE TITLE
[java] false positive in DataflowAnomalyAnalysis

### ISSUE THREAD

**Issue by Vampire** [pmd/pmd#1670]
**Affects PMD Version:** *6.11.0*
**Rule:** *DataflowAnomalyAnalysis*
**Description:**
Maybe I just don't understand the intention of the rule, but I don't see how the code below should trigger `Found 'UR'-anomaly for variable 'throwable' (lines '80'-'83').`

**Code Sample demonstrating the issue:**

```
public void send(String logMessage) {
    webhookClient.send(logMessage)
            .exceptionally(throwable -> {
                logger.error("Caught unhandled exception!", unwrapThrowable(throwable));
                return null;
            });
}
```

**Running PMD through:** *Gradle*

**Comment by jsotuyod** [comment]
Thanks for the report. This is a duplicate of #1636, we plan to stop looking for UR anomalies altogether
